src/server/services/validate-upload.js

```diff
@@ -43,6 +43,13 @@ async function validateEcCode ({ upload, records, trns }) {
   const coverSheet = records.find(doc => doc.type === 'cover').content
   const codeString = coverSheet['Detailed Expenditure Category']
 
+  if (codeString === undefined) {
```

Comment:
is there a reason to specifically check for `undefined` ? would `if(!codeString)` work?

Reply:
ah good call. The falsy evaluation is nicer because it will catch empty strings and nulls too. updating now